st2common/st2common/hooks.py

```diff
@@ -13,16 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import datetime
+import urlparse
```

Comment:
I thought so as well, but then I grepped code for `urlparse` and I didn't see it using six.moves, so I thought maybe I remember it wrong that this has been moved in Python 3, lol.

Anyway, yeah, you are right, it has indeed been moved. Will also update other code.